scripts/install_ebpf.psm1

```diff
@@ -137,14 +168,8 @@ function Stop-eBPFComponents {
     }
     # Stop the drivers and services.
     $EbpfDrivers.GetEnumerator() | ForEach-Object {
-        try {
```

Comment:
This used to have a try around Stop-Service.  Is it no longer needed around new line 135?